config/migrations/2024/20241108111400-change-rangorde-TS-to-schepen.sparql

```diff
@@ -0,0 +1,33 @@
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX bestuursfunctieCode: <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/>
+
+DELETE {
+  GRAPH ?graph {
+  ?mandataris mandaat:rangorde ?rangorde.
+  }
+}
+INSERT {
+  GRAPH ?graph {
+    ?mandataris mandaat:rangorde ?newLabel.
+  }
+}
+WHERE {
+  GRAPH ?graph {
+    ?mandataris a mandaat:Mandataris.
+    ?mandataris org:holds ?mandaat.
+    ?mandataris mandaat:rangorde ?rangorde.
+
+
+    VALUES ?schepenMandaat {
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e000014 # MANDAAT_SCHEPEN_CODE
+      bestuursfunctieCode:59a90e03-4f22-4bb9-8c91-132618db4b38 # MANDAAT_TOEGEVOEGDE_SCHEPEN_CODE
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e00001e # MANDAAT_DISTRICT_SCHEPEN_CODE
+      bestuursfunctieCode:5ab0e9b8a3b2ca7c5e000020 # MANDAAT_GEDEPUTEERDE_CODE
```

Comment:
https://github.com/lblod/frontend-lokaal-mandatenbeheer/blob/master/app/models/mandaat.js#L62

Reply:
let's remove gedeputeerde here. I will create a ticket to fix the rank for them too